### PR TITLE
[master][KOGITO-1770] - Renaming examples and README links

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

See: https://issues.redhat.com/browse/KOGITO-1770

Small fix for the examples renaming

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,11 @@ Getting Started
 
 This module contains a number of examples that you can take a look at and try out yourself.
```
Comment (jstastny-cz) [inline README.md:38]:
```suggestion
The [Kogito Examples repository](https://github.com/kiegroup/kogito-examples) contains a number of examples that you can take a look at and try out yourself.
```

**Comment by ricardozanini** [comment]
@jstastny-cz fixed, thanks!

**Comment by spolti** [comment]
Is this one ready for merge?

**Comment by ricardozanini** [comment]
@jstastny-cz ^

**Pull request merged**